plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlClient.java

```diff
@@ -164,15 +188,28 @@
     // An empty character means that the table doesn't have a comment in MySQL
     private static final String NO_COMMENT = "";
 
+    private static final JsonCodec<ColumnHistogram> HISTOGRAM_CODEC = jsonCodec(ColumnHistogram.class);
+
+    // We don't know null fraction, but having no null fraction will make CBO useless. Assume some arbitrary value.
+    private static final Estimate UNKNOWN_NULL_FRACTION_REPLACEMENT = Estimate.of(0.1);
```

Comment:
cc: @sopel39 in case there's a better value we can pick (maybe some prior work or literature)?

0.1 makes cost based join pushdown a bit problematic (and 10% is a very large assumption for null values IMO) since the null-fraction impacts join output size a lot making it difficult to configure the automatic join pushdown limits.

Reply:
This is for a follow-up, not needs to be done here.

Reply:
cc @raunaqmorarka any comment here?

Reply:
BTW it has been like this for 3 years in Starburst MySQL connector, and no-one really complained. I am going to merge this as is, but obviously we can improve later.

Reply:
I wouldn't worry about 10% differences in estimates, there's plenty of bigger simplifying assumptions in the CBO.